lib/wrapper.js

```diff
@@ -26,6 +29,13 @@ class WebSocketServerWrapper extends WebSocketServerChannel {
 
 		// When a WebSocket connects to the server...
 		this.server.on("connection", (socket, req) => {
+			// Set up heartbeat ("pong" listener) on raw socket, if needed
+			if(this.options.heartbeatInterval > 0) {
+				this._pong = true;
```

Comment:
`this._pong = true;`

should be

`socket._pong = true;`

Reply:
Oops!  Thank you.